client/index.html

```diff
@@ -1,54 +1,117 @@
-<!doctype html>
+<!DOCTYPE html>
 <html>
   <head>
-    <meta charset="utf-8">
+    <meta charset="utf-8" />
     <title>Commonwealth</title>
-    <link rel="stylesheet" href="/static/fontello/css/fontello.css" />
-    <link rel="stylesheet" href="/static/fontello/css/animation.css" />
     <link rel="stylesheet" href="/static/fonts/calibre.css" />
-    <link rel="preload" href="/static/fonts/Calibre-Semibold.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/static/fonts/Calibre-SemiboldItalic.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/static/fonts/Calibre-Medium.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/static/fonts/Calibre-MediumItalic.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/static/fonts/Calibre-Regular.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/static/fonts/Calibre-RegularItalic.woff2" as="font" crossorigin="anonymous" />
-    <link rel="preload" href="/static/fontello/font/fontello.woff2?80376133" as="font" crossorigin="anonymous" />
-    <link rel="icon" href="/favicon.ico"><!-- 32×32 -->
-    <link rel="icon" href="/favicon.svg" type="image/svg+xml">
-    <link rel="apple-touch-icon" href="/180.png"><!-- 180×180 -->
-    <link rel="manifest" href="/manifest.webmanifest">
-    <meta name="title" content="Commonwealth">
-    <meta name="description" content="Discuss, organize, and grow decentralized communities">
-    <meta name="author" content="">
-    <meta name="twitter:card" content="summary">
-    <meta name="twitter:title" content="Commonwealth">
-    <meta name="twitter:description" content="Discuss, organize, and grow decentralized communities">
-    <meta name="twitter:image:src" content="https://commonwealth.im/static/img/logo.png">
+    <link
+      rel="preload"
+      href="/static/fonts/Calibre-Semibold.woff2"
+      as="font"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="preload"
```

Comment:
I know the linter wants this, but I don't. Gonna push back on linting all of these <link> tags